.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "IIS-Sandbox"]
+	path = IIS-Sandbox
+	url = https://github.com/dvci/IIS-Sandbox.git
+[submodule "hl7v2-fhir-vaccine-credential-converter"]
```

Comment:
Would specify a tagged release branch here for the converter since the express API URL changes with each release, we don't want to always point to head. I think you can do this by adding something like below. May also want to do the same with the iis-sandbox in case of breaking release changes.
`branch = tags/v0.2.0`